tools/export/uvision5.py

```diff
@@ -28,7 +28,7 @@ class Uvision5(Exporter):
     Exporter class for uvision5. This class uses project generator.
     """
     # These 2 are currently for exporters backward compatiblity
-    NAME = 'uVision5'
+    NAME = 'uvision5'
```

Comment:
what targets use uv4 that dont use progen. This may be a good chance to push all uv support there